ebodac/src/test/java/org/motechproject/ebodac/uitest/test/BookingApplicationRescheduleVisitUiTest.java

```diff
@@ -0,0 +1,53 @@
+package org.motechproject.ebodac.uitest.test;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.motech.page.LoginPage;
+import org.motech.test.TestBase;
+import org.motechproject.ebodac.uitest.page.*;
```

Comment:
Try not to use wildcard import, they are bad for maintainability of code. You can force intellij not to do it
http://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Checkstyle rule about this:
http://checkstyle.sourceforge.net/config_imports.html#AvoidStarImport